src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetJudicialTaskControllerCFTTest.java

```diff
@@ -77,26 +72,26 @@ public void should_return_a_200_with_allocateHearingJudge_task_and_standard_gran
             .body("task.case_id", notNullValue())
             .body("task.case_category", equalTo("Protection"))
             .body("task.case_name", notNullValue())
-            .body("task.auto_assigned", notNullValue())
+            .body("task.auto_assigned", equalTo(true))
             .body("task.warnings", notNullValue())
-            .body("task.permissions.values", hasItems("Read", "Refer", "Execute"))
+            .body("task.permissions.values", hasItems("Read", "Refer", "Own"))
             .body("task.permissions.values", hasSize(3))
             .body("task.description", notNullValue())
-            .body("task.role_category", equalTo("ADMIN"));
+            .body("task.role_category", equalTo("JUDICIAL"));
 
         common.cleanUpTask(taskId);
     }
 
-    private void initiateTaskForJudicial(TestVariables taskVariables) {
+    private void initiateTaskForJudicial(TestVariables taskVariables, String taskType, String taskName) {
 
         ZonedDateTime createdDate = ZonedDateTime.now();
         String formattedCreatedDate = CAMUNDA_DATA_TIME_FORMATTER.format(createdDate);
         ZonedDateTime dueDate = createdDate.plusDays(1);
         String formattedDueDate = CAMUNDA_DATA_TIME_FORMATTER.format(dueDate);
 
         InitiateTaskRequest req = new InitiateTaskRequest(INITIATION, asList(
-            new TaskAttribute(TASK_TYPE, "allocateHearingJudge"),
-            new TaskAttribute(TASK_NAME, "Allocate hearing judge"),
+            new TaskAttribute(TASK_TYPE, taskType),
+            new TaskAttribute(TASK_NAME, taskName),
```

Comment:
Can we include IT's to cover the other 2 AC's?

1. If auto-assignable is true + authorisation is empty = assign task
2. If auto-assignable is true + authorisation has value -> check authorisation, if no match = do not assign task